py/dml/g_backend.py

```diff
@@ -39,8 +39,12 @@ def enc(expr):
             val = expr.value
             if isinstance(val, list):
                 return [enc(e) for e in val]
-            else:
+            elif isinstance(val, (int, str, bytes, float, bool)):
                 return val
+            else:
```

Comment:
Can refactor this slightly to discard the duplicate else-cases. (drop the else from the return on 49, and drop the entire else clause on 44.

Tbh, Im not sure if this is an anti-pattern (leads to less readable code, but also eliminates some code duplication)

Reply:
Fisketur suggests even more extreme refactorings which I _don't_ like, but I don't mind reducing the duplicate else-cases.